dev/tests/static/testsuite/Magento/Test/Integrity/DependencyTest.php

```diff
@@ -783,14 +784,16 @@ function (&$moduleName) {
     public function collectRedundant()
     {
         $schemaDependencyProvider = new DeclarativeSchemaDependencyProvider();
+        $graphQlSchemaDependencyProvider = new GraphQlSchemaDependencyProvider();
```

Comment:
is it necessary to use "new" here? don't know if you have access to object manager here..

Reply:
I think yes, all objects in this file are created by "new".